.github/workflows/workflow.yaml

```diff
@@ -57,6 +57,6 @@ jobs:
         run: yarn prepublish
       - name: Release
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
```

Comment:
i also added a secret with this title [here](https://github.com/fiatconnect/fiatconnect-types/settings/secrets/actions)